site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -61,6 +63,92 @@ What properties can be retrieved?
+| ``dnsResolutionStartTimeMicros``         | when resolving a domain name started, in microseconds since the   |
+|                                          | epoch (01-Jan-1970 00:00:00 UTC)                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``dnsResolutionDurationNanos``           | the duration took to resolve a domain name                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectStartTimeMicros``         | when connecting to a remote peer started, in microseconds since   |
+|                                          | the epoch (01-Jan-1970 00:00:00 UTC)                              |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectDurationNanos``           | the duration took to connect to a remote peer                     |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionStartTimeMicros``    | when waiting the completion of an existing connection attempt     |
+|                                          | started, in microseconds since the epoch                          |
+|                                          | (01-Jan-1970 00:00:00 UTC)                                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionDurationNanos``      | the duration took to wait the completion of an existing connection|
+|                                          | attempt to use one connection in HTTP/2                           |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionStartTimeMicros`` | when acquiring a connection started, in microseconds since the    |
+|                                          | epoch (01-Jan-1970 00:00:00 UTC)                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionDurationNanos``   | the duration took to get a connection                             |
++------------------------------------------+-------------------------------------------------------------------+
+
+Client connection timing properties
+-----------------------------------
+
+There are four properties in client connection timings. ``connectionAcquisitionDurationNanos`` is the sum of
+rest of the events which are represented by ``dnsResolutionDurationNanos``, ``socketConnectDurationNanos`` and
+``pendingAcquisitionDurationNanos``. Each event will or won't be occurred depending on the circumstance.
```

Comment:
- will or won't be occurred -> may or may not occur? (I don't know but it sounds better to me.)
- depending on circumstances (no the, plural)